contracts/src/main/kotlin/com/contractsAndStates/states/GameBoardState.kt

```diff
@@ -42,21 +57,26 @@ data class GameBoardState(val hexTiles: MutableList<HexTile>,
 class HexTile(val resourceType: String,
```

Comment:
@rogersanick Any reason why it is not a `data class`?

Reply:
None imparticular! We can definitely convert it to a data class if required